Practice/Practice/packages.config

```diff
@@ -11,4 +12,6 @@
   <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="2.0.0" targetFramework="net472" />
   <package id="Microsoft.jQuery.Unobtrusive.Validation" version="3.2.11" targetFramework="net472" />
   <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net472" />
+  <package id="Modernizr" version="2.8.3" targetFramework="net472" />
+  <package id="System.Linq.Dynamic" version="1.0.7" targetFramework="net472" />
```

Comment:
Po co w projekcie webowym Linq.Dynamic? Wszystkie akcje na obiektach powinny być wykonywane w Repository

Reply:
To znaczy wcześniej tego nie wiedziałem, ale już to poprawiłem.

Reply:
Ok, można merge'ować z master!